Assets/Scripts/Spawning/SpawnScript.cs

```diff
@@ -21,7 +21,15 @@ public HashSet<GameObject> SpawnEnemies()
             if (enemyToSpawn != null)
             {
                 enemyToSpawn.tag = "Enemy";
-                GameObject spawnedEnemy = Instantiate(enemyToSpawn, spawnPoint.transform.position, Quaternion.identity);
+                GameObject spawnedEnemy = Instantiate(enemyToSpawn, spawnPoint.transform.position, 
```

Comment:
Cool stuff, so you can specify attributes for a spawn point and an enemy will automatically "inherit" these?

Reply:
Yep, each spawn point now lets you set not only the enemy to spawn but also what attributes it should come with :)